docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -9,24 +9,46 @@ ifdef::notBuildingForSite[]
 include::menu.adoc[]
 endif::notBuildingForSite[]
 
+:imagesdir: ../../../../images/design/clustered
+
 == Introduction
 
 To enable Clustering with Terracotta, firstly you will have to start the http://terracotta.org[Terracotta server] configured with clustered storage.
 And then need a cache manager with clustering capabilities so that the caches can consume the clustered storage.
 
+Detailed instructions on how to configure and start a Terracotta server array can be found elsewhere.
+In this document we will use the minimalist configuration and setup for the server that is relevant in the context of this document.
+
+[[start-server]]
 == Starting Terracotta server
 
+You can start the server with the following configuration.
+It has the bare minimum configuration required for the samples in the rest of the document to work.
+
+[source,xml]
+----
+include::{sourcedir}/clustered/client/src/test/resources/configs/docs/tc-config.xml[]
+----
+
+The above configuration defines two named _server off-heap resources_: *primary-server-resource* and *secondary-server-resource*.
```

Comment:
And again, the italic is ok, the bold is wrong - you are referencing something from a source or config - use backticks for that